src/app/pages/about/about.html

```diff
@@ -41,15 +41,16 @@ <h3 class="ion-padding-top ion-padding-start">Details</h3>
           <ion-select-option value="seattle">Seattle, WA</ion-select-option>
         </ion-select>
       </ion-item>
-      <ion-item>
+      <ion-item button="true" id="open-date-input">
         <ion-label>
           Date
         </ion-label>
-        <ion-datetime
-          displayFormat="MMM DD, YYYY"
-          max="2056"
-          [(ngModel)]="conferenceDate">
-        </ion-datetime>
+        <ion-text slot="end">{{ conferenceDate | date: 'mediumDate' }}</ion-text>
+        <ion-popover id="date-input-popover" trigger="open-date-input" show-backdrop="false" side="top" alignment="end">
```

Comment:
This will fail if using Angular's strict template styling as it expects a boolean for `showBackdrop` so we need to use bindings here.

```suggestion
        <ion-popover id="date-input-popover" trigger="open-date-input" [showBackdrop]="false" side="top" alignment="end">
```